eve/main.yml

```diff
@@ -225,22 +225,33 @@ stages:
           dockerfile: docs/Dockerfile
     steps:
     - Git: *git_pull
+    - EveProperty:
+        name: Setup documentation builddir
+        property: documentation_builddir
+        value: "%(prop:builddir)s/build/artifacts/documentation"
     - ShellCommand:
         name: 'Build doc'
         haltOnFailure: true
         command: |-
-          tox --workdir /tmp/tox -e docs -- html BUILDDIR="%(prop:builddir)s/build/artifacts"
+          tox --workdir /tmp/tox -e docs -- html BUILDDIR="%(prop:documentation_builddir)s"
           # Building the latex twice to catch up the reference
-          tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:builddir)s/build/artifacts"
-          tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:builddir)s/build/artifacts"
+          tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:documentation_builddir)s"
+          tox --workdir /tmp/tox -e docs -- latexpdf BUILDDIR="%(prop:documentation_builddir)s"
         workdir: build/docs
-
+    - ShellCommand:
+        name: Create tar file for documentation
+        command: >-
+          tar --create -gzip
```

Comment:
I think you are missing one dash: should be `--gzip`

Reply:
True, good catch, now I wonder how it worked :D